backtracepython/attributes/report_data_builder.py

```diff
@@ -0,0 +1,20 @@
+import unicodedata
+
+
+class ReportDataBuilder:
+    primitive_types = (int, float, str, bool, bytes, type(None))
+
+    @staticmethod
+    def get(provider_attributes):
+        attributes = {}
+        annotations = {}
+
+        # Iterate through input_dict and split based on value types
+        for key, value in provider_attributes.items():
+            if isinstance(value, ReportDataBuilder.primitive_types):
+                attributes[key] = value
+            else:
+                annotations[key] = value
+
+        # Return both dictionaries
+        return attributes, annotations
```

Comment:
not sure why this is a class? why just not a function?

Reply:
I want to move the more into an OOP pattern. I'm just sticking to the plan

Reply:
well, static functions is not OOP, that's why I don't see the point